### DISCUSSION TITLE
Allure report is showing wrong time taken for execution of suite

### DISCUSSION THREAD

**Discussion by rajeshallure** [allure-framework/allure2#2654] [Questions & Support]
If i run 3 scripts on 3 days for 1 min each, allure report is showing total time taken as 3 days, whereas actual execution time is 3 min. How can i correct this?

**Comment by baev** [comment]
As for now, it shows the time between the first test start and the last test stop. It's often misleading, but I don't know how to make it clearer. We're considering replacing it with the report generation timestamp.
